packages/node/tsconfig.json

```diff
@@ -1,17 +1,18 @@
 {
   "compilerOptions": {
     "strict": true,
+    "allowJs": true,
     "esModuleInterop": true,
-    "lib": ["ES2020", "DOM"],
-    "target": "ES2020",
-    "module": "commonjs",
+    "lib": ["ES2021", "DOM"],
```

Comment:
This will drop support for node 14 which we can't do until it reaches EOL in April

Reply:
@cb1kenobi can you land this change with `ES2020` or does this change have to wait on Node 14 EOL from Amazon?

Reply:
Today I tested using Node.js 14 with both `lib` and `target` set to `ES2021` and `vc dev` worked. In theory it should have not worked. Tomorrow I'll set them to `ES2020` and test again with Node 14 and 16.